test/index.js

```diff
@@ -1247,6 +1267,29 @@ describe('function helpers', () => {
   })
 
   describe('config', () => {
+    describe('dependency tracing', () => {
+      it('extracts a list of all dependencies', async () => {
+        await moveNextDist()
+        await nextRuntime.onBuild(defaultArgs)
+        const dependencies = await getAllPageDependencies(constants.PUBLISH_DIR)
+        expect(dependencies.map((dep) => relative(process.cwd(), dep))).toMatchSnapshot()
+      })
+
+      it('extracts dependencies that exist', async () => {
+        await moveNextDist()
+        await nextRuntime.onBuild(defaultArgs)
+        const dependencies = await getAllPageDependencies(constants.PUBLISH_DIR)
+        const filesExist = await Promise.all(
+          dependencies.map(async (dep) => {
+            const exists = await pathExists(dep)
+            return [dep, exists]
+          }),
+        )
+        console.log(filesExist)
+        expect(filesExist.every(([dep, exists]) => exists)).toBeTruthy()
```

Comment:
Is returning the `dep` necessary in the tuple?

```suggestion
        const filesExist = await Promise.all(
          dependencies.map(async (dep) => {
            const exists = await pathExists(dep)
            return exists
          }),
        )

        expect(filesExist.every((exists) => exists)).toBeTruthy()
```

Reply:
Oh crap. Good spot. I put that in for debugging.